<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tw">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>481</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Поиск</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество предметов в создании</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="nums_craft">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Название прдметов в создании</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Писать через &quot;;&quot;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Станция создания</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Любая</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Инвентарь</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Размещаемость</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>207</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Предмет можно разместить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Предмет нельзя разместить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>Не важно можно или нет</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>182</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Теги</string>
   </property>
  </widget>
  <widget class="QComboBox" name="teg1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ничего</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Блок</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Еда</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Наживка</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Валюта</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Броня</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Маг</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Существо</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Фоновая стена</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Воин</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Призыв босса</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Боеприпасы</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Осветительный прибор</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Станок</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Оружие</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Заражение</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Аксессуар</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="teg2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ничего</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Блок</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Еда</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Наживка</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Валюта</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Броня</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Маг</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Существо</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Фоновая стена</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Воин</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Призыв босса</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Боеприпасы</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Осветительный прибор</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Станок</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Оружие</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Заражение</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Аксессуар</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="teg3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ничего</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Блок</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Еда</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Наживка</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Валюта</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Броня</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Маг</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Существо</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Фоновая стена</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Воин</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Призыв босса</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Боеприпасы</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Осветительный прибор</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Станок</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Оружие</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Заражение</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Аксессуар</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="b_or">
      <property name="text">
       <string>Теги работают как и</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="b_and">
      <property name="text">
       <string>Теги работают как или</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ВЫХОД В МЕНЮ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
